backend/Cargo.toml

```diff
@@ -37,6 +37,8 @@ tracing = "0.1.41"
 typst = "0.12.0"
 typst-pdf = "0.12.0"
 quick-xml = { version = "0.37.1", features = ["serialize"] }
+sha2 = "0.10.8"
+const-hex = "1.14.0"
```

Comment:
Can you elaborate why you chose `const-hex`? It looks like it relies on some [unsafe Rust](https://github.com/search?q=repo%3ADaniPopes%2Fconst-hex%20unsafe&type=code) for performance, but we don't need it to be fast. 

The [hex crate](https://crates.io/crates/hex) is already in our dependency tree and it doesn't look like that one uses unsafe Rust.

Reply:
My main reason for not using hex is that it is no longer maintained and appears to be as good as abandoned. Alternatively there is also [faster-hex](https://crates.io/crates/faster-hex), but that one also has some unsafe usages.

Reply:
Hmm I just noticed some new commits on the repository, maybe it will come back to life again?

Reply:
As discussed on mattermost: switched to `hex` in cf5ada3fd988d9e4a4b5e0e57d9cb0f316752845